it/v1/endpoints/AmendBenefitAmountsControllerISpecIntegrationBaseSpec.scala

```diff
@@ -21,13 +21,12 @@ import play.api.http.HeaderNames.ACCEPT
 import play.api.http.Status._
 import play.api.libs.json.{JsObject, JsValue, Json}
 import play.api.libs.ws.{WSRequest, WSResponse}
-import support.IntegrationBaseSpec
+import support.V1IntegrationBaseSpec
 import v1.models.errors._
 import v1.stubs.{AuditStub, AuthStub, DesStub, MtdIdLookupStub}
 
-class AmendBenefitAmountsControllerISpec extends IntegrationBaseSpec {
+class AmendBenefitAmountsControllerISpecIntegrationBaseSpec extends V1IntegrationBaseSpec {
```

Comment:
Don't think that (and other IT classes) should have been renamed. Leave as `AmendBenefitAmountsControllerISpec` etc  etc

Reply:
this was an accident, I didn't realise this had even been changed